Mathlib/SetTheory/Cardinal/Cofinality.lean

```diff
@@ -1114,6 +1117,37 @@ theorem sum_lt_of_isRegular {ι : Type u} {f : ι → Cardinal} {c : Cardinal} (
   sum_lt_lift_of_isRegular.{u, u} hc (by rwa [lift_id])
 #align cardinal.sum_lt_of_is_regular Cardinal.sum_lt_of_isRegular
 
+@[simp]
+theorem iUnion_subset_cardinal_lt {ι : Type u} {α : Type u} {t : ι → Set α} {c : Cardinal} :
+    #(⋃ i, t i) < c → ∀ i, #(t i) < c :=
+ fun h _ =>  lt_of_le_of_lt (Cardinal.mk_le_mk_of_subset <| subset_iUnion _ _) h
```

Comment:
```suggestion
theorem card_lt_of_card_iUnion_lt {ι α : Type u} {t : ι → Set α} {c : Cardinal}
    (h : #(⋃ i, t i) < c) (i : ι) : #(t i) < c :=
 lt_of_le_of_lt (Cardinal.mk_le_mk_of_subset <| subset_iUnion _ _) h
```

Reply:
As above, this is an example of our [Hypotheses Left of Colon](https://leanprover-community.github.io/contribute/style.html#hypotheses-left-of-colon) style.

Reply:
I see that my suggested rename got dropped. Was this intentional? (It is possible there is a naming convention in this corner of the library with which I am not familiar.)

Reply:
I’m not sure how I missed that, sorry. I’ll change that!

Reply:
It's easier for me to leave this unresolved until the change is in. Feel free to resolve after you have pushed the commit.